eventmodel.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE "tags" (
+  "id" integer PRIMARY KEY,
```

Comment:
Minor: to stay homogeneous with the team, use uppercase keys 'INTEGER', 'VARCHAR'...